recipes/songbird/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.8.2" %}
+
+package:
+  name: songbird
+  version: {{ version }}
+
+source:
+  fn: songbird-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/songbird/songbird-{{ version }}.tar.gz
+  sha256: 1e152e5d6114135d0299a362ffd082ca525a6b4f8fc0cc1891ea131892e10a37
+
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --record=record.txt
+
+requirements:
+  build:
+    - python >=3.5
+    - pandas
+    - click
+    - tensorflow >1.7
+    - numpy >1.13
+    - biom-format
+    - scikit-bio >0.5.4
+    - seaborn
+    - patsy
+    - scikit-learn
+    - tqdm
+
+  run:
+    - python >=3.5
+    - pandas
+    - click
+    - tensorflow >1.7
+    - numpy >1.13
+    - biom-format
+    - scikit-bio >0.5.4
+    - seaborn
+    - patsy
+    - scikit-learn
+    - tqdm
+
+test:
+  imports:
+    - songbird
+
+
+about:
+  home: https://github.com/biocore/songbird
+  license: BSD 3-Clause
```

Comment:
Please also add license family and file, like at https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L62